build-system/get-dep-graph.js

```diff
@@ -137,6 +137,9 @@ exports.getBundleFlags = function(g) {
   const indexOfAmp = bundleKeys.indexOf('src/amp.js');
   bundleKeys.splice(indexOfAmp, 1);
   bundleKeys.splice(1, 0, 'src/amp.js');
+  const indexOfIntermedia = bundleKeys.indexOf('_base_i');
```

Comment:
i did the splice thing as a hack, but im guessing we'd want some sort of comparator on this at some point?

Reply:
Yeah, well, a lot of code here needs some love :)